2017-2018/botCode/pacbotCommsModule.py

```diff
@@ -40,7 +40,7 @@ def __init__(self, server_addr, server_port, local_addr, local_port):
 
     def msg_received(self, msg, msg_type):
         # This gets called whenever any message is received
-        return 
+        return
```

Comment:
Lol what

Reply:
removed a trailing space. this is fine. Cleans up the code